hack/run-local.sh

```diff
@@ -14,5 +15,5 @@ echo "Image: ${IMAGE}"
 echo "Release version: ${RELEASE_VERSION}"
 echo "Namespace: ${NAMESPACE}"
 
-${DELVE:-} ./ingress-operator start --image "${IMAGE}" --release-version "${RELEASE_VERSION}" \
+${DELVE:-} ./ingress-operator start --image "${IMAGE}" --canary-image "${CANARY_IMAGE}" --release-version "${RELEASE_VERSION}" \
```

Comment:
According to [this](https://github.com/openshift/cluster-ingress-operator/pull/476/files#diff-24ce9f24fd9e7e47b7ac69fa0b597fe0470c1ba7b8257d0fa6537c9caf1b5deeR64), `canary-image` is optional. If this is the case, the flag should be set optionally in the `hack/*` scripts.

Reply:
Good point. I've updated the `hack/run-local.sh` script accordingly.